pages/linux/useradd.md

```diff
@@ -17,3 +17,7 @@
 - Create new user with supplementary groups (mind the lack of whitespace):
```

Comment:
Suggestion - "with supplementary groups" -> "belonging to additional groups" for simplicity.